Procfile

```diff
@@ -1 +1 @@
-release: npm i -g fastly-cli && fastly purge-all -k $FASTLY_PURGE_KEY -s $FASTLY_SERVICE_ID
+release: if [ -z "$FASTLY_PURGE_KEY" ]; then echo "Skipping Fastly purge because FASTLY_PURGE_KEY is not set"; else curl -XPOST -H "Fastly-Key:$FASTLY_PURGE_KEY" https://api.fastly.com/service/$FASTLY_SERVICE_ID/purge_all; fi
```

Comment:
can we add a check on the curl output? Because that's the only way for us to know that the fastly cache cleared before actual deployment takes place.

Reply:
@sivakumar-kailasam sorry for the _noob_ question but... what do you mean by "add a check on the curl output"? do you have an example?

Reply:
We need to be sure that the request was successful. Sorry but I've never had to do things like this with curl before. I would've asked for the npm command to be added back if that was possible at all.

Reply:
@sivakumar-kailasam I'm pretty sure my latest change should meet your requirements 👍